Разметка главной страницы

eggsandbaky opened the pull request:
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1808931/tasks/3)

:boom: https://htmlacademy-htmlcss.github.io/1808931-sedona-33/2/

eggsandbaky committed:
Настроил editconfig

Взял конфигурацию из кодгайда

eggsandbaky committed:
Установил lint html и stylelint

eggsandbaky committed:
Создал страницу каталога

Создал страницу каталога /catalog.html

eggsandbaky committed:
Разметил главную страницу

Задание 2.34.
Разметка главной страницы проекта Sedona.

maxzbest commented on index.html:
```diff
@@ -5,8 +5,139 @@
+      </nav>
+    </header>
+    <main class="main-index">
+      <h1 class="visually-hidden">Добро пожаловать в Седону!</h1>
```
Давай здесь напишем что-нибудь более информативное, например "Седона. Сайт туристического городка в штате Аризона."

maxzbest commented on index.html:
```diff
@@ -5,8 +5,139 @@
+            <p class="adventages-number">— №1 —</p>
+            <p class="adventages-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+            <img class="adventages-photo" src="" alt="Фотография города Седона">
+            <ul class="adventages-list town-list">
```
Получается, что у тебя этот элемент списка отличается от остальных, при такой схеме, соответственно нарушается консистентность / однородность списка

Вынеси его отдельно, или вообще всё сделай сделай элементами одного списка

eggsandbaky replied:
Заменил ol на ul, но оставил вложенную структуру списка

eggsandbaky committed:
Исправил h1 заголовок

Сделал заголовок h1 более информативным.

eggsandbaky committed:
Заменил список

Заменил нумерованный список преимуществ на маркированный

eggsandbaky committed:
Оттипографировал(?) текст

Прогнал большие не скрытые текстовые элементы через типограф
https://www.artlebedev.ru/typograf/
